### DISCUSSION TITLE
update on the extension remover fix

### DISCUSSION THREAD

**Discussion by Theblackmamba824** [3kh0/ext-remover#167] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
any updates on the ext remover
if u reply just say if its close or not close to being fixed

**Comment by 3kh0** [comment, marked as answer]
I am not in a position to report on that.

**Reply by synapt0** [reply]
Why not?

**Reply by Theblackmamba824** [reply]
> I am not in a position to report on that.

so who is

**Reply by 3kh0** [reply]
@Theblackmamba824 The people in the TN discord, I did not create this exploit

**Reply by ghost** [reply]
@3kh0 which discord?

**Reply by 3kh0** [reply]
titanium network discord, discord.gg/unblock. bypassi created the exploit
